Userland/Libraries/LibGfx/ImageFormats/GIFLoader.cpp

```diff
@@ -159,7 +160,8 @@ static ErrorOr<void> decode_frame(GIFLoadingContext& context, size_t frame_index
         auto const previous_image_disposal_method = i > 0 ? context.images.at(i - 1)->disposal_method : GIFImageDescriptor::DisposalMethod::None;
 
         if (i == 0) {
-            context.frame_buffer->fill(Color::Transparent);
+            Gfx::PainterSkia painter(*context.frame_buffer);
```

Comment:
Could this be `Gfx::Painter` amd `Gfx::Painter::create`? Would match other usages